main.py

```diff
@@ -0,0 +1,100 @@
+import ee
+import argparse
+import geemap
+from geopy.geocoders import Nominatim
+from geopy.exc import GeopyError
+from decouple import config
+
+HEAT_MAP_GEE_PROJECT = config("GEE_PROJECT")
+ee.Initialize(project=HEAT_MAP_GEE_PROJECT)
+
+def get_city_coordinates()->list:
+    """Asks user for city"""
+    try: 
+        city_name = str(input("Enter the city urban center you want to view: "))
+        geolocator = Nominatim(user_agent="urban_heat_island")
+
+        location = geolocator.geocode(city_name)
+        if location:
+            city_coordinates = [location.latitude, location.longitude]
+            print(f"The center coordinates for your city are {city_coordinates}")
+            return city_coordinates
+        else:
+            raise ValueError(f"Location for '{city_name}' not found.")
+    
+    except GeopyError as e:
```

Comment:
should this try/except be narrower? what is the line that can raise this exception, I guess `Nominatim` right?